main/mpg123/APKBUILD

```diff
@@ -20,7 +18,6 @@ build() {
 		--host=$CHOST \
 		--prefix=/usr \
 		--disable-dependency-tracking \
-		--with-ipv6 \
```

Comment:
why?

Reply:
it appears to be on by default

Reply:
I'll add that info to the commit message